requirements.txt

```
@@ -3,3 +3,5 @@ flake8-annotations==2.9.1
 flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
 pep8-naming==0.13.2
+
+requests~=2.28.1
```

```suggestion
requests==2.28.1
```
And please remove blank line before